sdk/src/offline-key-provider.ts

```diff
@@ -106,21 +126,21 @@ class OfflineSearchParams implements KeySearchParams {
     }
 
     /**
-     * Create a new OfflineSearchParams instance for the transfer_public_to_private function of the credits.aleo program.
+     * Create a new OfflineSearchParams instance for the transfer_public_as_signer function of the credits.aleo program.
      */
-    static transferPublicToPrivateKeyParams(): OfflineSearchParams {
-        return new OfflineSearchParams(CREDITS_PROGRAM_KEYS.transfer_public_to_private.locator, true);
+    static transferPublicAsSignerKeyParams(): OfflineSearchParams {
```

Comment:
In the sense of usual software yes, but the SDK tracks API changes of the Aleo network which will likely start major version updates post mainnet, so I'd like to continue to track the Network's conventions here. However, it does deserve at least a minor version upgrade.

Reply:
The SDK is currently on version `0.6.9`. So a major version bump would mean going to `0.7.0`.

This is because the `0.x.x` versions are special, and so a minor version bump is treated the same as a major version bump:

https://www.npmjs.com/package/semver

> Many authors treat a `0.x` version as if the `x` were the major "breaking-change" indicator.
>
> Caret ranges are ideal when an author may make breaking changes between `0.2.4` and `0.3.0` releases, which is a common practice. However, it presumes that there will not be breaking changes between `0.2.4` and `0.2.5`.

Reply:
Ah yes, you're right, a major version bump is in order. It was bumped in the latest commit.